ls1intum/Artemis
# `Development`: Update Apollon version

### Checklist
#### General



### Motivation and Context
Update version of Apollon from [v2.11.3](https://github.com/ls1intum/Apollon/releases/tag/v2.11.3) to [v2.12.1](https://github.com/ls1intum/Apollon/releases/tag/v2.12.1) [(Changelog)](https://github.com/ls1intum/Apollon/compare/v2.11.3...v2.12.1)

### Description
This bugfix release version of Apollon implements the manual layouting feature in Apollon along with minor bugfixes ([changelog](https://github.com/ls1intum/Apollon/releases/tag/v2.12.1)).

### Steps for testing
1. Create a modeling exercise.
2. Insert couple of elements and connect them in a way that the connectors have more than 3 edges. 
3. Observe that the connection can now be layouted manually with the presence of manual layout handle.


### Screenshots
![screen-capture (21)](https://user-images.githubusercontent.com/14681902/185104388-4d76aa9f-67d8-4c24-8c26-bbddfe5483c8.gif)


##### Changelog: https://github.com/ls1intum/Apollon/compare/v2.11.3...v2.12.1
